Authentication/Configuration/ColorPalette.swift

```diff
@@ -21,14 +21,18 @@ public protocol ColorPaletteType {
     var background: UIColor { get }
     
     var links: UIColor { get }
+    var labels: UIColor { get }
+    var textfieldText: UIColor { get }
+    var passwordVisibilityButtonText: UIColor { get }
+    var mainButtonText: UIColor { get }
     
 }
 
 public extension ColorPaletteType {
     
-    public var logInButtonDisabled: UIColor { return UIColor(hexString: "#d8d8d8ff")! }
-    public var logInButtonEnabled: UIColor { return UIColor(hexString: "#f5a623ff")! }
-    public var logInButtonExecuted: UIColor { return UIColor(hexString: "#e78f00ff")! }
+    public var mainButtonDisabled: UIColor { return UIColor(hexString: "#d8d8d8ff")! }
+    public var mainButtonEnabled: UIColor { return UIColor(hexString: "#f5a623ff")! }
+    public var mainButtonExecuted: UIColor { return UIColor(hexString: "#e78f00ff")! }
```

Comment:
shouldn't we extract this to a protocol for buttons? (which declares a color for each button state)?